tools/RELEASE.md

```diff
@@ -114,13 +117,13 @@ An alternative is to push branch and tags separately:
     $ git push --dry-run --tags origin v0.11.4rc1
 
 
-## 5. Wait for CI builds
+## 6. Wait for CI builds
 
 Monitor travis-ci builds by looking atthe *tag* build at
 [travis-ci]https://travis-ci.org/confluentinc/confluent-kafka-python
```

Comment:
markdown link format is:
`[linktext](url)`

Reply:
Not sure I understand. Are you requesting an additional improvement on the docs?

Reply:
the link to travis-ci is incorrect, check the rendered version of RELEASE.md and you'll see the link doesnt work. So use the above format.